builtin/mainmenu/dlg_create_world.lua

```diff
@@ -15,6 +15,12 @@
 --with this program; if not, write to the Free Software Foundation, Inc.,
 --51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 
+-- load default settings
+core.setting_set("mg_flags","trees, caves")
+core.setting_set("mgv5_spflags","blobs")
+core.setting_set("mgv6_spflags","biomeblend, jungles, mudflow")
+core.setting_set("mgv7_spflags","mountains, ridges")
```

Comment:
We have to load the default, so any player will create world without worrying about the parameter.